transport-native-unix-common/src/main/java/io/netty/channel/unix/ServerVSockChannel.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2025 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.channel.unix;
+
+import io.netty.channel.ServerChannel;
+
+public interface ServerVSockChannel extends ServerChannel {
```

Comment:
javadoc

Reply:
should probably also override the local/remoteAddress methods with specific return types